packages/npm/@amazeelabs/gatsby-source-silverback/src/helpers/schema.test.ts

```diff
@@ -1,4 +1,5 @@
 import { directives } from '@amazeelabs/test-directives';
+import type { Gat } from 'gatsby/dist/schema/types';
```

Comment:
Oh my Gat! What's that? 😄 

Do we have static tests turned off for test files on purpose? Or why static test didn't catch this?

Reply:
Indeed, tests where excluded. Fixed that.